### PR TITLE
{vis}[system/system] Panoply v5.5.1

### PR DESCRIPTION
**Pull request by WilleBell**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by WilleBell** [comment]
Test report by @WilleBell
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node601.hydra.os - Linux Rocky Linux 8.10, x86_64, AMD EPYC 9384X 32-Core Processor, Python 3.6.8
See https://gist.github.com/WilleBell/330a8c91c2f032332586b0ffe21a5d4d for a full test report.

File: easybuild/easyconfigs/p/Panoply/Panoply-5.5.1.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+postinstallcmds = [
+    'mkdir %(installdir)s/bin',
+    'mv %(installdir)s/panoply.sh %(installdir)s/bin',
+    'sed -i "s,jars,../jars,g" %(installdir)s/bin/panoply.sh',
+    'ln -s %(installdir)s/bin/panoply.sh %(installdir)s/bin/panoply',
+]
+
+sanity_check_paths = {
+    'files': ['bin/panoply'],
```
Comment (boegel) [inline easybuild/easyconfigs/p/Panoply/Panoply-5.5.1.eb:28]: @WilleBell Can we add a sanity check command for this? Like `panoply --help` or something?

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/p/Panoply/Panoply-5.5.1.eb (hunk 1 above)
Comment (WilleBell) [reply easybuild/easyconfigs/p/Panoply/Panoply-5.5.1.eb:28]: I couldn't find anything that would serve as sanity check, it all opens the GUI.

Comment (boegel) [reply easybuild/easyconfigs/p/Panoply/Panoply-5.5.1.eb:28]: OK, nevermind then

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3123.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/a703e521db7c07c6b78b406291a24550 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen3

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Going in, thanks @WilleBell!

**Pull request merged**